README.md

```diff
@@ -302,7 +302,10 @@ module.exports = {
 
 ### `sassOptions`
 
-Type: `object | (content: string|Buffer, loaderContext: LoaderContext, meta: any) => object `
+Type: 
+```ts
+object | (content: string|Buffer, loaderContext: LoaderContext, meta: any) => object 
```

Comment:
Highlight is not working, because syntax is broken, let's use:
```
type sassOptions = object | (content: string|Buffer, loaderContext: LoaderContext, meta: any) => object
```